syntax/odin.vim

```diff
@@ -139,7 +140,8 @@ highlight link odinUnion Structure
 highlight link odinBitField Structure
 highlight link odinBitSet Structure
 
-highlight link odinFunction Function
+highlight link odinFunctionDecl Function
+highlight link odinFunctionCall Function
```

Comment:
Since not everyone will want function highlighting, maybe we should group the `highlight link` and the `syntax match` for it together and comment them out by default.

We can then add a section above the image in the readme about it:
```md
### Off-by-default features

- Function call highlighting
  Enable by uncommenting the two adjacent lines marked with `:FunctionHighlighting` in `syntax/odin.vim`.
```
And in the syntax file:
```vim
" Uncomment to enable :FunctionHighlighting
"syntax match odinFunctionCall "\v\w+\s*(\()@="
"highlight link odinFunctionCall Function
```

Thoughts?